scripts/remote-tmux-live-fuzz.sh

```
@@ -252,16 +254,61 @@ check_screen_oracle() {
   # census: every window with an on-screen pane is the VISIBLE window, so tmux's
   # full pane list for it must be on screen. A missing pane is the app failing to
   # render a pane of the visible window, which is a defect, not less coverage.
+  # One workspace presents one tmux window at a time, so surfaces from two
+  # windows on screen at once is a hidden tab drawing over the selected one.
+  # The per-window census below cannot see this: each window individually
+  # matches tmux while both are visible.
+  on_screen_windows=$(printf '%s' "$PANE_SURFACES_JSON" | jq -r '
+    [.panes[]? | select(.on_screen == true) | .window_id] | unique | join(" ")
+  ' 2>/dev/null)
+  if [ "$(printf '%s\n' $on_screen_windows | grep -c .)" -gt 1 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)